R/covariate_analysis_panel.R

```diff
@@ -20,14 +17,15 @@ covariate_analysis_panel_ui <- function(id) {
           selectInput(
             inputId = ns("covariate_type_selection"),
             label = "",
-            choices = c("binary", "continuous")
+            choices = c("Binary", "Continuous"),
+            width = "120pt"
           )
         ),
         style = "display: inline-block;"
       ),
       div(
         conditionalPanel(
-          condition = "output.inferred_type == 'continuous'",
+          condition = "output.inferred_type == 'Continuous'",
```

Comment:
I'm confused about the logic behind this message "If your data is binary, the only allowed values are 0, 1, and NA"

Reply:
I've changed the wording to hopefully make it clearer. Happy to workshop it if you think that it still needs it.

Reply:
> I've changed the wording to hopefully make it clearer. Happy to workshop it if you think that it still needs it.

That's much clearer